cli/endpoints/online/triton/multi-models/multi-endpoint-aks.yml

```diff
@@ -0,0 +1,21 @@
+name: aks-multi-model
+type: online
+target: azureml:aks-gpu-deploy
+auth_mode: Key
+traffic:
+    etblue: 100
+deployments:
+    - name: etblue
+      model: azureml:multi-models:2
+      code_configuration:
+          code:
+             local_path: ./code
+          scoring_script: dummy.py
+      environment:
+            name: aks-multi-env
```

Comment:
Environment depends on the image.  Same Environment is used for aks, mir.   Remove 'aks' or 'mir' from environment names as that will be misleading for the user.

Reply:
Done as per comments